src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -147,6 +160,8 @@ protected AbstractEntityAIStructure(@NotNull final J job)
            */
           new AITarget(COMPLETE_BUILD, this::completeBuild, STANDARD_DELAY)
         );
+
+        decoItems = BlockTags.getCollection().getOrCreate(new ResourceLocation(MOD_ID, DECOBLOCK_TAG));
```

Comment:
This is not the way to do tags.

Reply:
Ok, I've updated this to static, to use a better pattern for the caching of this. Given that this is the way all of our tags access is in 1.15, I think it should stay consistent here and be updated to be 'right' in 1.16 using the TagUtils class.